content/spin/dynamic-configuration.md

```diff
@@ -179,3 +180,50 @@ url = "redis://localhost"
 ```
 
 You must individually grant each component access to the stores that it needs to use. To do this, use the `component.key_value_stores` entry in the component manifest within `spin.toml`. See [Spin Key Value Store](kv-store-api-guide.md) for more details. 
+
+### SQLite Storage Runtime Configuration
+
+Spin provides built-in SQLite storage. By default, this is backed by a database that Spin creates for you. However, you can use the Spin runtime configuration file (`runtime-config.toml`) to add and customize SQLite databases.
+
+The following example `runtime-config.toml` tells Spin to map the `default` database to an SQLite database elsewhere in the file system:
+
+```toml
+[sqlite_database.default]
+path = "/planning/todo.db"
+```
+
+If you need more than one database, you can configure multiple databases, each with its own name:
+
+```toml
+# This defines a new store named todo
+[sqlite_database.todo]
+path = "/planning/todo.db"
+
+# This defines a new store named finance
+[sqlite_database.finance]
+path = "/super/secret/monies.db"
+```
+
+Spin creates any database files that don't exist.
+
+Spin can also use [libSQL](https://libsql.org/) databases accessed over HTTPS.  libSQL is fully compatible with SQLite but provides additional features including remote, distributed databases.
+
+> Spin does not provide libSQL access to file-based databases, only databases served over HTTPS.
+
+To use libSQL, set `type = "libsql"` in your `runtime-config.toml` entry.  You must then provide a `url` and authentication `token` instead of a file path.  For example, this entry tells Spin to map the `default` database to a libSQL service running on `libsql.example.com`:
+
+```toml
+# This tells Spin to use the remote host as its default database
+[sqlite_database.default]
+type = "libsql"
+url = "sensational-penguin-ahacker.libsql.example.com"
+token = "a secret"
+```
+
+Spin does _not_ create libSQL databases.  Use your hosting service's tools, or the `libsql` command line, to create them.  You can still set up tables and data in a libSQL database via `spin up --sqlite`.
+
+> Some libSQL service documentation shows URLs with a `libsql://` scheme.  Don't include a scheme in the `runtime-config.toml` `url` field.  Include only the host name.
+
+The `default` database will still be defined, even if you add other databases.
+
+By default, components will not have access to any of these databases (even the default one). You must grant each component access to the databases that it needs to use. To do this, use the `component.sqlite_databases` entry in the component manifest within `spin.toml`. See [SQLite Database](/spin/sqlite-api-guide.md) for more details. 
```

Comment:
I'm not sure if this is the right article, but we should instruct users on how to delete their DB if they so desire.  Perhaps that's just adding a few sentences informing them they'll have to manually delete the directory.

Reply:
I've added a sentence about custom databases, which is what this page covers.  But were you referring to the default database?